dashboard-for-desktop/articles/dashboard-designer/provide-data/binding-to-federated-data-source.md

```diff
@@ -5,72 +5,77 @@ author: Margarita Zakhodyaeva
 
 # Binding to Federated Data Source
 
-A _federated data source_ combines multiple data sources (except OLAP) in one.
+You can create a federated data source if a dashboard contains at least one data source. Create a federated query where you specify how to federate existing data sources.
 
-## Overview
+You can create federated queries of the following types:
 
-You can create a federated data source if a dashboard contains at least one data source. To do this, create a query where you specify how to federate existing data sources. The federated query can be one of the following types:
+- **Join**
 
-### Join
-Combines rows from two or more tables based on a column they share. The join type specifies records that have matching values in both tables.
+  Combines rows from two or more tables based on a clause. 
 
-### Union
-Combines rows from two or more tables into one data set and removes duplicate rows in merged tables. You can only create a union query for data sources that contain columns with the same name. Data types of such columns should be [implicitly converted](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/types/casting-and-type-conversions#implicit-conversions).
+- **Union and Union All**
```

Comment:
Is this the new section title (to replace `### Union`)? If so, the style (`**Union and Union All**`) is different from `### Union`. Is that okay?